fix at_time notebook links

jli opened the pull request:
Note: the "source lag" link was replaced with a link to the FAQ doc in our API developer docs, but there wasn't a corresponding question in that doc for the 2nd link: "how do I see previous values fo a time series point to see how the value changed over time".

If this functionality is still possible, we should probably add it to the current FAQ doc?

muzigao commented:
@jli  the second point is the "show_revisions" flag in from https://developers.gro-intelligence.com/api.html#groclient.GroClient.get_data_points

@JGrinst   do you remember if we had "how do I see previous values fo a time series point to see how the value changed over time" in FAQ before? was that removed on purpose?

JGrinst commented:
<p>We deleted the github wiki, so that we wouldn't have documentation in two different places. But I believe that specific FAQ is now covered by https://developers.gro-intelligence.com/Recipes/querying-data.html#show-revisions</p>

Commit:
fix at_time notebook links

- The "source lag" wiki link was replaced with a link to the FAQ doc in
  our API developer docs (fixed in #296)
- The "how do I see previous values for a time series point to see how
  the value changed over time" link was replaced with a link to our
  docs for the show_revisions option for get_data_points. (removed in
  #296, adding back correct replacement link)

jli commented:
Thanks Justin! I replaced the 2nd wiki link with that link for `show_revisions`.

Muzi, please take another look.
